console/console-init/ui/src/modules/address-space/dialogs/CreateMessagingProject/MessagingProjectReview.tsx

```diff
@@ -8,22 +8,26 @@ import { Loading } from "use-patternfly";
 import { useQuery } from "@apollo/react-hooks";
 import { ADDRESS_SPACE_COMMAND_REVIEW_DETAIL } from "graphql-module/queries";
 import { AddressSpaceReview } from "modules/address-space/components";
+import { IMessagingProject } from "./CreateMessagingProject";
 
-export interface IReviewProps {
-  name?: string;
-  type?: string;
-  plan?: string;
-  namespace: string;
-  authenticationService: string;
+interface IMessagingProjectReviewProps {
+  projectDetail: IMessagingProject;
 }
 
-export const Review: React.FunctionComponent<IReviewProps> = ({
-  name,
-  type,
-  plan,
-  namespace,
-  authenticationService
+export const MessagingProjectReview: React.FunctionComponent<IMessagingProjectReviewProps> = ({
+  projectDetail
 }) => {
+  const {
+    name,
+    namespace,
+    type,
+    plan,
+    authService,
+    protocols,
+    customizeEndpoint,
+    addRoutes,
+    tlsCertificate
+  } = projectDetail || {};
   const { data, loading } = useQuery(ADDRESS_SPACE_COMMAND_REVIEW_DETAIL, {
     variables: {
       as: {
```

Comment:
Endpoints need to be added, so the 'addressSpaceCommand' can include them, for display in the 'configuration details'

Reply:
@vbusch I updated the variable to add endpoints. It seems mockserver doesn't provide xml with endpoint configuration